pyproject.toml

```diff
@@ -1,27 +1,3 @@
-[tool.black]
-py311 = true
-include = '\.pyi?$'
-exclude = '''
-/(
-    \.git
-  | \.mypy_cache
-  | \.tox
-  | \.venv
-  | build
-  | dist
-  | node_modules
-)/
-'''
-
-# settings to make isort compatible with how black formats imports
-[tool.isort]
```

Comment:
We were not running `isort`. Ruff does sort imports. Those settings are under `[tool.ruff.lint.isort]` if we want them.